app/controllers/ctc/questions/w2s_controller.rb

```diff
@@ -12,6 +12,16 @@ def self.show?(intake)
         benefits_eligibility.claiming_and_qualified_for_eitc?
       end
 
+      def add_w2_later
+        analytics_journey = AnalyticsJourney.find_or_initialize_by(client: current_intake.client)
```

Comment:
Safest is to use `create_or_find_by` per [this article](https://sikac.hu/use-create-or-find-by-to-avoid-race-condition-in-rails-6-0-f44fca97d16b).